### ISSUE TITLE
Word by word keyboard navigation in chat input doesn't work

### ISSUE THREAD

**Issue by torarnv** [charmbracelet/crush#641]
In my terminal I've set up navigation between words so I can move between words and delete word by word via zsh config:

```
❯ cat ~/.zsh/subword.zsh
# Adds subword navigation to the ZSH line editor

#unset WORDCHARS

autoload -U select-word-style
#select-word-style whitespace

autoload -U forward-word-match backward-word-match kill-word-match backward-kill-word-match
zle -N forward-subword forward-word-match
zle -N backward-subword backward-word-match
zle -N kill-subword kill-word-match
zle -N backward-kill-subword backward-kill-word-match

zstyle ':zle:*-subword*' word-style normal-subword
zstyle ':zle:*-subword*' word-chars '*?.[]~=/&;!#$%^(){}<>'

bindkey '\e[1;5C' forward-subword
bindkey '\e[1;5D' backward-subword

bindkey '\e[1;7C' kill-subword
bindkey '\e[1;7D' backward-kill-subword


# https://unix.stackexchange.com/questions/258656/how-can-i-delete-to-a-slash-or-a-word-in-zsh
```

But in the Crush chat input this doesn't work. Please support the same bindings that have been already set in the terminal (whatever they are), so users get a consistent UX in their input fields. Thanks!
